CIP-1694/README.md

```diff
@@ -0,0 +1,1161 @@
+---
+CIP: 1694
+Title: A First Step Towards On-Chain Decentralized Governance
```

Comment:
@JaredCorduan I was also wondering that 😎 ... and would vote for setting the PR title to the reviewed document title because of the requirement that CIP titles be technically descriptive.

Reply:
I also prefer "A First Step Towards On-Chain Decentralized Governance".